application/src/main/java/org/togetherjava/tjbot/commands/free/UserStrings.java

```diff
@@ -0,0 +1,62 @@
+package org.togetherjava.tjbot.commands.free;
+
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nullable;
+
+/**
+ * Class containing all the strings sent to users during their interaction with the free command
+ * system. This does not include the logged strings or the exception strings.
+ */
+public enum UserStrings {
+    NEW_QUESTION("""
+            Thank you for asking a question in an available channel.
+            When a helper who can answer this question reads it they will help you.
+            Please be patient.
+
+            __Please do not post your question in other channels__
+            """),
+    MARK_AS_FREE("""
+            This channel is now free for a question to be asked.
+            """),
+    ALREADY_FREE_ERROR("""
+            This channel is already free, no changes made
+            """),
+    NOT_READY_ERROR("""
+            Command not ready please try again in a minute
+            """),
+    NOT_MONITORED_ERROR("""
+            This channel is not being monitored for free/busy status. If you believe this channel should be part of the free/busy status system, please discuss it with a moderator
```

Comment:
```suggestion
            This channel is not being monitored for free/busy status.
            If you believe this channel should be part of the free/busy status system, please discuss it with a moderator
```
Seems like we had a miscommunication here, I was talking about this one specifically.
Most likely my mistake for phrasing it poorly.

I'm fine with having all of them as text blocks.
Just looking at it, it kinda does look more readable? But a small difference so whatever you prefer.

Reply:
for most of them text block vs standard is not much change
but `NEW_QUESTION` should be a text block (its pre formatted code)
and `NOT_MONITORED_ERROR` should NOT be a text block? its currently all the way to `column 180` because discord must word wrap it dynamically.

other than the insanely long lines if its left as a text block i have no preference either way

Reply:
I'm missing why we're forcing Discord to do the error one?

Reply:
it just happens to be long. but its a single paragraph
if we pre-format it it will display weirdly (on cellphones for example)

the other long string is NEW_QUESTION but thats completely separate points, they need to start on differant lines.

its the differance between:
Hello
I am pleased to meet you.
Can I please have your name.
Do you like music?

and:
The police
are at the door and
need to be let in,
can you open the
door for them, please.